airflow/models/baseoperator.py

```diff
@@ -1183,17 +1186,17 @@ def render_template_fields(
         context: Context,
         jinja_env: Optional["jinja2.Environment"] = None,
     ) -> Optional["BaseOperator"]:
-        """Template all attributes listed in template_fields.
+        """Template all attributes listed in *self.template_fields*.
 
         This mutates the attributes in-place and is irreversible.
 
-        :param context: Dict with values to apply on content
-        :param jinja_env: Jinja environment
+        :param context: Context dict with values to apply on content.
+        :param jinja_env: Jinja environment to use for rendering.
         """
         if not jinja_env:
             jinja_env = self.get_template_env()
         self._do_render_template_fields(self, self.template_fields, context, jinja_env, set())
-        return self
+        return None
```

Comment:
This does not change any behaviour, I only did it so the behaviour is easier to explain…